force-app/main/default/classes/FeedComment_TriggerHandler.cls

```diff
@@ -0,0 +1,98 @@
+/**                     **************                    ***
+ * @description       : Helper class for FeedComment Trigger
+ * @author            : Deepak Prajapati (d.prajapati@concret.io)
+ * @last modified on  : 05-10-2021
+ * @last modified by  : Deepak Prajapati (d.prajapati@concret.io)
+**************************************************************/
+public with sharing class FeedComment_TriggerHandler {
+
+    Public static final String ORG_URL = URL.getSalesforceBaseUrl().toExternalForm();
+    
+    /* TEST VARIABLE */
+    @TestVisible public static List<String> testEmails;
+
+    /****************
+    * @description Method for sending email to Account Followers
+    * @author Deepak Prajapati (d.prajapati@concret.io) | 28-09-2021 
+    * @param List<FeedItem> feedItemList --> Trigger.New  
+    * @param String operationType  --> Either Insert or Update
+    ***************************************/
+    public static void FeedComment_Trigger(List<FeedComment> feedComments, String operationType) {
+
+        /* Get All Parent Account Ids  */
+        Set<Id> parentIds = new Set<Id>();
+        Set<Id> parentFeeds = new Set<Id>();
+        
+        for(FeedComment comment : feedComments) {
+            if ( !parentIds.contains( comment.parentId ) ) {
+                parentIds.add(comment.ParentId);
+            }
+            parentFeeds.add(comment.FeedItemId);
+        }
+
+        Map<Id, Account> feedParentAccounts = new Map<Id, Account> ([  SELECT 
+                                                                            Name,
+                                                                            Id,
+                                                                            CreatedBy.FirstName,
+                                                                            CreatedBy.LastName 
+                                                                        FROM Account 
+                                                                        WHERE id IN :parentIds]);
+
+        Map<Id, FeedItem> feedItemsMap = new Map<Id, FeedItem>([SELECT 
+                                                                    Id, 
+                                                                    type 
+                                                                FROM FeedItem 
+                                                                WHERE id IN: parentFeeds]);
```

Comment:
use consistency if you're going to use one line per field approach then use this in all of your queries.